orchagent/fgnhgorch.cpp

```diff
@@ -113,6 +147,13 @@ void FgNhgOrch::setStateDbRouteEntry(const IpPrefix &ipPrefix, uint32_t index, N
 
 }
 
+void FgNhgOrch::removeStateDbRouteEntry(const string& ipPrefix)
```

Comment:
>removeStateDbRouteEntry [](start = 16, length = 23)

Better to remove the function and just use `del()` in the original caller.

Reply:
removed the function removeStateDbRouteEntry